fastplotlib/graphics/line.py

```diff
@@ -24,7 +25,7 @@ def __init__(self, data: np.ndarray, zlevel: float = None, size: float = 2.0, co
             material=material(thickness=size, vertex_colors=True)
         )
 
-        self.events = {}
+        self.registered_callbacks = {}
```

Comment:
set this in the base `Graphic`, and use dict() to create a new dict (there are caveats to initializing dicts and lists with expressions like this that can lead to weird bugs)